src/issues/issue-2/david-byrne/index.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react"
+import Img from "gatsby-image"
+import { useStaticQuery, graphql } from "gatsby"
+import { useMediaQuery } from "react-responsive"
+
+import Header from "./Header"
+import AnimatedWords from "./AnimatedWords"
+import Pagination from "./Pagination"
+
+import useMedia from "./useMedia"
+import useMeasure from "./useMeasure"
+import usePositions from "./usePositions"
+import albumByYearData from "./albumsByYear.js"
+import { mapImagesToAlbums, calculateGridItems } from "./utility"
+import {
+  Background,
+  Text,
+  FlexContainer,
+  ImgWrap,
+  WordCountContainer,
+} from "./styled"
+import { breakpoints } from "../../../styles/layout"
+
+const xOffset = window.innerWidth / 2
+
+const DavidByrne = () => {
+  const isPhoneWide = useMediaQuery({
+    query: breakpoints.phoneWide,
+  })
+  // Query images
+  const { mini, big } = useStaticQuery(graphql`
+    query {
+      mini: allFile(
+        filter: { relativeDirectory: { eq: "issue2-images/david-byrne/mini" } }
+      ) {
+        ...ImageFragment
```

Comment:
I used this fragment to keep this code block small, definition can be found `./types.js`. Note, if you define a fragment, you do **not** need to import it into the component file, Gatsby will do this for you automagically. 

> When compiling your site, Gatsby preprocesses all GraphQL queries it finds. Therefore, any file that gets included in your project can define a snippet. However, only Pages can define GraphQL queries that actually return data. This is why you can define the fragment in the component file - it doesn’t actually return any data directly.

documentation here: https://www.gatsbyjs.org/docs/using-graphql-fragments/